.circleci/config.yml

```diff
@@ -4,7 +4,8 @@ jobs:
   started:
     resource_class: small
     docker:
-    - image: 907128454492.dkr.ecr.us-east-1.amazonaws.com/betterment/notifier:latest
+    - image: 907128454492.dkr.ecr.us-east-1.amazonaws.com/betterment/coach-notifier/generic:1
+    working_directory: "~/journaled"
```

Comment:
@kelvin-acosta what is this `working_directory` config?

Reply:
this is generated by coach CLI, right? if so, I think the answer is "don't worry about it" :-D

Reply:
yeah this is generated via coach CLI - i only ask because I originally ran the `coach update ci` command in a directory with a custom name and it pulled that name into the config

Reply:
which seemed wrong haha